semgrep/semgrep/rule.py

```diff
@@ -55,7 +55,7 @@ def _parse_boolean_expression(
                         )
                     else:
                         raise InvalidRuleSchemaError(
-                            f"operator {operator} must have children"
+                            f"operator {boolean_operator} must have children"
```

Comment:
Small cleanup here, this was something like `and_inside`, now it's `pattern-inside`.